core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -461,7 +461,7 @@ angular.module('oppia').directive('questionsList', [
             var associatedSkillSummaries = ctrl.associatedSkillSummaries;
             var newQuestionIsBeingCreated = ctrl.newQuestionIsBeingCreated;
             var categorizedSkills = ctrl.getSkillsCategorizedByTopics;
-            var untriagedSkillSummaries = ctrl.untriagedSkillSummaries;
+            var untriagedSkillSummaries = ctrl.getUntriagedSkillSummaries;
```

Comment:
Just a note -- something seems wrong here, because the get() should be a function. I think they are declared wrongly at the top -- should be using '&' rather than '=' perhaps for one-way binding.

OK to fix separately, but could you please file an issue? Make sure the issue is very clear and fully enumerates the steps of what to do. Check with one of the LaCE team leads whether they understand it and can fix it.

Thanks!

Reply:
@seanlip filed #10124. Can you ptal at the issue? I'm not sure if I've understood the issue correctly. (As far as I've understood the names of the props are invalid. It looks like they are functions but are not. So, they need to be renamed. Is that correct?)

Reply:
Thanks @nishantwrp, I think it suffices.  I left an additional comment that might help.

There are two possible fixes, basically. One is to change the names since they're not functions. But the other is to change the binding, so that the inner code only reads from the outer code and there's no two-way syncing. That latter option might be better if the binding doesn't actually need to be two-way (one-way binding is more efficient).